manifests/g/GoLang/Go/Unstable/1.19.0/GoLang.Go.Unstable.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GoLang.Go.Unstable
+PackageVersion: 1.19.0
+PackageLocale: en-US
+Publisher: https://golang.org
+PublisherUrl: https://golang.org/
+PublisherSupportUrl: https://golang.org/help
+PrivacyUrl: http://www.google.com/intl/en/policies/privacy/
+# Author: 
+PackageName: Go Programming Language
```

```suggestion
PackageName: Go Programming Language (Unstable)
```
Just so that it doesn't conflict with the stable version.